INFO  2020-08-16 18:29:09,144 261511ms SalesPriceRepository   GetPriceOfItem     - URL AbsoluteUri = http://localhost:8080/SalesPrices/GetPriceOfItem
INFO  2020-08-16 18:29:09,301 261668ms SalesPriceRepository   GetPriceOfItem     - URL Parameters = ?org_id=1&bus_part_id=18&date=2020/08/16&item_id=25&uom_id=7&qty=1.00
ERROR 2020-08-16 18:29:09,446 261813ms SalesPriceRepository   GetPriceOfItem     - Failed
System.AggregateException: One or more errors occurred. ---> System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: The connection was closed unexpectedly.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at MyERP.Web.SalesPriceRepository.GetPriceOfItem(Int64 OrgId, Int64 BusPartId, DateTime date, Int64 ItemId, Int64 UomId, Decimal Qty) in c:\Projects\MyERP-WEB\MyERP.Web\Repository\SalesPriceRepository.cs:line 37
---> (Inner Exception #0) System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: The connection was closed unexpectedly.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---<---

INFO  2020-09-09 10:54:11,239 125257ms SalesPriceRepository   GetPriceOfItem     - URL AbsoluteUri = http://localhost:8080/SalesPrices/GetPriceOfItem
INFO  2020-09-09 10:54:11,598 125616ms SalesPriceRepository   GetPriceOfItem     - URL Parameters = ?org_id=1&bus_part_id=18&date=2020/09/09&item_id=19&uom_id=39&qty=1.00
ERROR 2020-09-09 10:54:12,871 126889ms SalesPriceRepository   GetPriceOfItem     - Failed
System.AggregateException: One or more errors occurred. ---> System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: The connection was closed unexpectedly.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at MyERP.Web.SalesPriceRepository.GetPriceOfItem(Int64 OrgId, Int64 BusPartId, DateTime date, Int64 ItemId, Int64 UomId, Decimal Qty) in c:\Projects\MyERP-WEB\MyERP.Web\Repository\SalesPriceRepository.cs:line 37
---> (Inner Exception #0) System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: The connection was closed unexpectedly.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---<---

